package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "gitbook-plugin-plantuml-cloud",
+  "name": "plantuml",
```

Comment:
Why did you rename the package name?
As far as I know, gitbook-plugin-xxxx is the naming convention of gitbook plugin.